### ISSUE TITLE
plupload flickers, then removes multiple files. Files uploaded, but not associated.

### ISSUE THREAD

**Issue by pd-cm** [pods-framework/pods#7138]
### Description

When selecting multiple files with plupload in frontend or admin, all files show as uploading for a moment, but then the field reverts to display only the first selected file.

If the form is submitted without attempting to upload again, all files get uploaded to the media library, but only the first image is associated with the related object (in this test case, a user).

If the user attempts to select images again, the images get uploaded to the Media Library multiple times, but again, only the first image selected in each select attempt gets displayed or associated.

So, if one selects images `1, 2, 3`, only `1` is displayed, but `1, 2, 3` get uploaded.
If before submitting, one again selects `2 & 3`, only `2` gets added to display, but `2 & 3` are uploaded again.
If before submitting, one again selects `3`, `3` gets added to display, but the upload now consists of `1, 2, 3, 2, 3, 3`, while the form only displays `1, 2, 3` and the association consists of `1, 2 (the second), 3 (the third)`

### Version

2.9.19

### Testing Instructions

The frontend shortcode in use was `[pods-form name="user" fields="upload_queue" where="id={@user.ID}" label="Upload Designs"]`

The effect is the same on frontend and admin.

### Screenshots / Screencast

![plupload-multiple-files](https://github.com/pods-framework/pods/assets/38669929/b80a184d-6843-4f8a-9d38-8b8a76c2d2fe)

### Possible Workaround

Adding/selecting files one at a time works fine, but if multiple images are selected, only the first appears in the upload field. If one of those images is then selected/uploaded again, it gets uploaded to the Media Library multiple times in one submission.

### Site Health Information

```text
Verified with LocalWP, WP 6.3.1, Pods 2.9.19.
```

See https://podswp.slack.com/archives/C02SVLHQF/p1692757636865369
See https://podswp.slack.com/archives/C02SVLHQF/p1693358610178719

### Pods Package

```json
{
    "@meta": {
        "version": "2.9.19",
        "build": 1693389028
    },
    "settings": {
        "types_only": "0",
        "watch_changed_fields": "0",
        "metadata_integration": "1",
        "metadata_override_get": "0",
        "media_modal_fields": "0",
        "session_auto_start": "0",
        "wisdom_opt_out": "1"
    },
    "pods": [
        {
            "name": "user",
            "id": 5,
            "label": "User",
            "description": "",
            "type": "user",
            "storage": "meta",
            "object": "user",
            "_migrated_28": "1",
            "required": "0",
            "required_help_boolean": "0",
            "groups": [
                {
                    "name": "more_fields",
                    "id": 6,
                    "label": "More Fields",
                    "description": "",
                    "weight": 0,
                    "fields": [
                        {
                            "name": "upload_queue",
                            "id": 7,
                            "label": "Upload Queue",
                            "description": "",
                            "weight": 0,
                            "type": "file",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "required_help_boolean": "0",
                            "file_format_type": "multi",
                            "file_uploader": "plupload",
                            "file_type": "images",
                            "file_attachment_tab": "upload",
                            "file_upload_dir": "wp",
                            "file_edit_title": "1",
                            "file_show_edit_link": "0",
                            "file_linked": "0",
                            "file_limit": "10",
                            "file_field_template": "rows",
                            "file_add_button": "Add File",
                            "file_modal_title": "Attach a file",
                            "file_modal_add_button": "Add File",
                            "file_wp_gallery_link": "file",
                            "file_wp_gallery_columns": "3",
                            "file_wp_gallery_size": "thumbnail",
                            "repeatable": "0",
                            "repeatable_format": "default",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        }
                    ]
                }
            ]
        }
    ]
}
```

**Comment by mpiasta-ca** [comment]
I also confirmed this issue with **plupload.** 

More details available on this WordPress Plugin support thread:

https://wordpress.org/support/topic/plupload-selecting-multiple-files-to-upload-only-first-is-uploaded/

**Comment by sc0ttkclark** [comment]
This bug really suck to fix, can't wait to refactor this area properly out of the Marionette solution in place right now.

Fixed in: f8ea339adf474316a38d962133ef856168d16bfc
